support/latest/policies/containerization.md

```diff
@@ -1,21 +1,19 @@
+![Containerized deployment]({% link support/images/deployment-support.png %})
 
-As with traditional on premise deployment methods, customers may wish to customize or build upon the technology and ideas within our deployment templates. The deployment templates provide a starting point for customers to simplify the creation of deployments unique to their own needs and skill sets.
+The above illustration shows four deployment options. Alfresco creates and supports only the software represented as App1, App2, and App3. Alfresco doesn’t create or support anything else in the diagram - that’s considered to be “infrastructure”. 
 
-Where customers make this kind of customization, Alfresco will support the underlying images, providing they are using the supported infrastructure components listed on our product supported stack configurations. The supported stacks can be found at:[https://www.alfresco.com/services/subscription/supported-platforms](https://www.alfresco.com/services/subscription/supported-platforms){:target="_blank"}.
+There are many different ways to deploy Alfresco software into an infrastructure before it can run. A few options include using custom shell scripts, installers, Ansible, Terraform, Docker Compose, Kubernetes, Docker Enterprise, OpenShift, Heroku, Chef, Puppet, Rancher, AWS CloudFormation, Azure Resource Manager, Google Cloud Deployment Manager, AWS ECS, AWS EKS, and so on.
 
-![Containerized deployment]({% link support/images/deployment-support.png %})
+Alfresco doesn’t create or support any deployment tool and customers may choose the deployment tool they feel most comfortable with. Often this will be based on a company standard, and Alfresco expects that each customer will have its own unique standard. Each customer is expected to have the skills and knowledge needed to use their chosen deployment tool effectively. Alfresco cannot provide assistance on the chosen deployment tool - there are simply too many for that to be feasible.
 
-The above illustration shows four deployment options. Alfresco creates and supports only the software represented as App1, App2, and App3. Alfresco doesn’t create or support anything else in the diagram - that’s considered to be “infrastructure”.
+From time to time Alfresco will create and make available reference deployment scripts to help people be successful with a couple of the more-popular deployment tools. These reference scripts are supported by Alfresco, and if a defect is found in them or an enhancement is wanted then a ticket can be raised through Alfresco Support. The customer is expected to know how to use the deployment technology associated with these scripts, and they shouldn’t use them if they don’t. Alfresco is unable to offer training in the use of deployment tools.
```

Comment:
more-popular - doesn't need a hyphen

Reply:
When used as a compound adjective it does, but I'll leave style to the UA team.